CMakeLists.txt

```
@@ -881,6 +884,16 @@ if(openPMD_INSTALL)
         install(TARGETS openPMD.py
             DESTINATION ${CMAKE_INSTALL_PYTHONDIR}/openpmd_api
         )
+        if(openPMD_BUILD_CLI_TOOLS)
+            foreach(toolname ${openPMD_PYTHON_CLI_TOOL_NAMES})
+                install(
+                    FILES ${openPMD_SOURCE_DIR}/src/cli/${toolname}.py
+                    DESTINATION ${CMAKE_INSTALL_BINDIR}
+                    RENAME openpmd-${toolname}.py
```

I would call the tool just `openpmd-pipe` because for the user it will make no difference if the tool is a script or an executable:
```suggestion
                    RENAME openpmd-${toolname}
```